src/app/identity.service.ts

```diff
@@ -0,0 +1,62 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Inject, Injectable, InjectionToken } from '@angular/core';
+import { TransferState } from '@angular/platform-browser';
+import { StorageService } from 'ngx-webstorage-service';
+import { Observable } from 'rxjs';
+
+import { AnalyticsService } from './analytics.service';
+import { environment } from '../environments/environment';
+import { Person } from './person.model';
+
+export const TBG_DONATE_ID_STORAGE = new InjectionToken<StorageService>('TBG_DONATE_ID_STORAGE');
+
+@Injectable({
+  providedIn: 'root',
+})
+export class IdentityService {
+  private readonly apiPath = '/people';
+  // Key is per-domain/env. For now we simply store a single JWT (or none).
+  private readonly storageKey = `${environment.identityApiPrefix}/v1/jwt`;
+
+  constructor(
+    private analyticsService: AnalyticsService,
+    private http: HttpClient,
+    @Inject(TBG_DONATE_ID_STORAGE) private storage: StorageService,
+    private state: TransferState,
+  ) {}
+
+  create(person: Person): Observable<Person> {
+    return this.http.post<Person>(
+      `${environment.identityApiPrefix}${this.apiPath}`,
+      person);
+  }
+
+  update(person: Person): Observable<Person> {
+    return this.http.put<Person>(
+      `${environment.donationsApiPrefix}${this.apiPath}/${person.id}`,
```

Comment:
@NoelLH You should be calling out to the Identity API here, not MatchBot

Reply:
Good catch!